types/splitio.d.ts

```diff
@@ -551,6 +560,9 @@ declare namespace SplitIO {
   interface GaToSplitIntegration {
     type: 'GA_TO_SPLIT',
   }
+  type SPLIT_IMPRESSION = 'IMPRESSION';
+  type SPLIT_EVENT = 'EVENT';
```

Comment:
Unless we scope these we'll need to rename them or move them where they're not exposed too.